src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java

```diff
@@ -241,15 +245,88 @@ private void ensureWithinRange(@NotNull Position position) throws TileOutOfRange
             throw new TileOutOfRangeException(position, xRange, yRange);
     }
 
+    private void setRunning(boolean newState){
```

Comment:
setRunningToFalse or similar.
it's not a classic setter, we can just set it to false.

Reply:
Changed it to setGameRunningToFalse, and did a check before the call. I think that makes more sense than that before.